packages/@react-aria/toast/stories/Example.tsx

```diff
@@ -43,13 +43,13 @@ function ToastRegion() {
 function Toast(props) {
   let state = useContext(ToastContext);
   let ref = useRef(null);
-  let {toastProps, titleProps, closeButtonProps} = useToast(props, state, ref);
+  let {toastProps, contentProps, titleProps, closeButtonProps} = useToast(props, state, ref);
   let buttonRef = useRef();
   let {buttonProps} = useButton(closeButtonProps, buttonRef);
 
   return (
     <div {...toastProps} ref={ref} style={{margin: 20, display: 'flex', gap: 5}}>
-      <div {...titleProps}>{props.toast.content}</div>
+      <div {...contentProps} {...titleProps}>{props.toast.content}</div>
```

Comment:
I guess you're not wanting to breaking change the API by renaming `titleProps` by adding the values of `contentProps` to it, but I'm not liking having two sets of props from the same origin go to the same element.

Reply:
I can wrap the `titleProps` within a separate `contentProps` div.